scheduler/filter/README.md

```diff
@@ -185,6 +185,18 @@ For example,
 * `constraint:node!=/foo\[bar\]/` will match all nodes, except `foo[bar]`. You can see the use of escape characters here.
 * `constraint:node==/(?i)node1/` will match node `node1` case-insensitive. So 'NoDe1' or 'NODE1' will also matched.
 
+#### Soft Affinities
+
+By default, affinities are hard enforced. If an affinity is not met, the container won't be scheduled.
+With soft affinites the scheduler will try to meet the affinity. If it is not met, the scheduler will discard the filter and schedule the container according to the scheduler strategy.
+
+soft affinites are expressed with a **~** in the expression
+Example ,
+```
+$ docker run -d --name redis1 -e affinity:image==~redis redis
```

Comment:
From the PR seems like you can also express a soft affinity to "not" place a node (!=~) somewhere. I think that's also worth documenting as that might not be obvious.